packages/editor/src/components/block-mover/drag-handle.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import BlockDraggable from '../block-draggable';
+
+export const IconDragHandle = ( { isVisible, className, icon, onDragStart, onDragEnd, blockElementId, order, rootClientId, clientId, layout } ) => {
```

Comment:
Actually, this is the component that I find "useless" because it has no meaning on its own :) I would fine to just call `<BlockDraggable` in the block mover component.

Reply:
I find this is useful to hide the drag logic from the block-mover itself, which shouldn't be concerned with _how_ it is implemented. It helps the readers understand the general UI without overwhelming them with implementation details.